remotes.go

```diff
@@ -99,6 +101,46 @@ func (r *remotesTable) WithProjectAndFilters(
 	return sql.NewSpanIter(span, iter), nil
 }
 
+// IndexKeyValueIter implements the sql.Indexable interface.
+func (*remotesTable) IndexKeyValueIter(
+	ctx *sql.Context,
+	colNames []string,
+) (sql.IndexKeyValueIter, error) {
+	s, ok := ctx.Session.(*Session)
+	if !ok || s == nil {
+		return nil, ErrInvalidGitbaseSession.New(ctx.Session)
+	}
+
+	iter, err := s.Pool.RepoIter()
+	if err != nil {
+		return nil, err
+	}
+
+	return &remotesKeyValueIter{repos: iter, columns: colNames}, nil
+}
+
+// WithProjectFiltersAndIndex implements sql.Indexable interface.
+func (*remotesTable) WithProjectFiltersAndIndex(
+	ctx *sql.Context,
+	columns, filters []sql.Expression,
+	index sql.IndexValueIter,
+) (sql.RowIter, error) {
+	span, ctx := ctx.Span("gitbase.ReferencesTable.WithProjectFiltersAndIndex")
```

Comment:
Shouldn't this be `"gitbase.RemotesTable.WithProjectFiltersAndIndex"`?

Reply:
Fixed